portfolio/src/main/webapp/script.js

```diff
@@ -39,4 +39,20 @@ function addRandomSoup() {
     //Add it to the page. 
     const soupContainer = document.getElementById('soup-container');
     soupContainer.innerText = soup;
+}
+
+function addRandom(lsts) {
+    const lst = lsts[Math.floor(Math.random() * lsts.length)];
+    return lst
+}
+
+/** Fetches the current date from the server and adds it to the page. */
+async function showServerString() {
+  const responseFromServer = await fetch('/hello');
+  const jsonFromResponse = await responseFromServer.json();
+
+  const helloContainer = document.getElementById('hello-container');
+  helloContainer.innerText = addRandom(jsonFromResponse);
+
+  console.log(jsonFromResponse);
```

Comment:
Can delete the `console.log` unless you need it for something else.

```suggestion
  const helloContainer = document.getElementById('hello-container');
  helloContainer.innerText = addRandom(jsonFromResponse);
```